clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -9018,7 +9018,7 @@ void SPIRVTranslator::ConstructJob(Compilation &C, const JobAction &JA,
     std::string DefaultExtArg =
         ",+SPV_EXT_shader_atomic_float_add,+SPV_EXT_shader_atomic_float_min_max"
         ",+SPV_KHR_no_integer_wrap_decoration,+SPV_KHR_float_controls"
-        ",+SPV_KHR_expect_assume";
+        ",+SPV_KHR_expect_assume,+SPV_KHR_linkonce_odr";
```

Comment:
To double check: the extension is being enabled on FPGA H/W as well intentionally? I don't mean that we shouldn't, just want to double check if we are sure that it will work and not break anything.

Reply:
Yeah, AFAIK it should be supported on FPGA as well. I've submitted a request to FPGA team and it was marked as implemented. I'm not sure it will be somehow used there though, since the main target was to use this extension for dynamic linking which won't be supported on FPGA because there is no runtime linking.